File: SEP3-TIER1/BlazorTest/Model/ProjectUser.cs
using Newtonsoft.Json;

namespace BlazorTest.Model
{
    public class ProjectUser
    {
        [JsonProperty("projectid", NullValueHandling = NullValueHandling.Ignore)]
        public int ProjectId { get; set; }
        [JsonProperty("project", NullValueHandling = NullValueHandling.Ignore)]
        public Project Project { get; set; }
        [JsonProperty("user", NullValueHandling = NullValueHandling.Ignore)]
        public User User { get; set; }
        [JsonProperty("username", NullValueHandling = NullValueHandling.Ignore)]
        public string Username { get; set; }

    }
}

File: SEP3-TIER2/src/Server/Model/ProjectUser.java
package Server.Model;

public class ProjectUser {

    private int projectid;
    private String username;

    public int getProjectid() {
        return projectid;
    }

    public void setProjectid(int projectid) {
        this.projectid = projectid;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    @Override
    public String toString() {
        return "ProjectUser{" +
                "projectid=" + projectid +
                ", username='" + username + '\'' +
                '}';
    }
}
